docs/hyscale-commands-reference.md

```diff
@@ -16,6 +16,7 @@ Options:
   -a, --application string   name of application `<applicationName>`
   -v  --verbose              `Optional` verbose mode
   -P  string                 `Optional` profile name 
+  -o, --output json          `Optional` After the command has completed its execution, deployment status will be shown in a json format.
```

Comment:
After the command has completed its execution, deployment status will be shown in a **requested** format. **Allowed values are json/JSON .**

Reply:
I think the explanation can be moved to the description part. Here we can say:
Show deployment status in the required format. Allowed values are json/JSON.